packages/frontend-core/src/api/user.js

```diff
@@ -296,9 +296,9 @@ export const buildUserEndpoints = API => ({
     })
   },
 
-  getTenantInfo: async ({ tenantId }) => {
+  getAccountHolder: async ({ appId }) => {
     return await API.get({
-      url: `/api/global/tenant/${tenantId}`,
+      url: `/api/global/users/holder/${appId}`,
```

Comment:
would make this `accountholder` or `accountowner` for consistency

Reply:
Renamed.